jax/_src/abstract_arrays.py

```diff
@@ -51,6 +51,13 @@ def canonical_concrete_aval(val, weak_type=None):
   return ConcreteArray(dtypes.canonicalize_dtype(np.result_type(val)), val,
                        weak_type=weak_type)
 
+def masked_array_error(*args, **kwargs):
+  raise ValueError("numpy masked arrays are not supported as direct inputs to JAX functions. "
```

Comment:
```suggestion
  raise ValueError("numpy masked arrays are not supported as direct inputs to `jax.numpy` functions. "
```
?

Reply:
I think the more general form is correct: it's not just `jax.numpy` functions, but *any* jax function (e.g. `jax.lax`, anything transformed with `jax.jit` or `jax.vmap`, etc.)